apps/users-service/test/handlers/users.create-romp.test.ts

```diff
@@ -0,0 +1,216 @@
+import Chance from 'chance';
+import nock from 'nock';
+import { handler } from '../../src/handlers/create-romp';
+import { apiGatewayEvent } from '../helpers/events';
+import { APIGatewayProxyResult } from 'aws-lambda';
+import { createRandomUser } from '../helpers/create-user';
+import { config as authConfig } from '@asap-hub/auth';
+
+jest.mock('@asap-hub/auth');
+
+const chance = new Chance();
+
+describe('GET /users/{id}/romp', () => {
+  let id, code;
+
+  const romp = {
```

Comment:
Should be typed as `const romp: CreateRomp = ...` to make sure properties are updated whenever we change the interface